### ISSUE TITLE
Lists / arrays

### ISSUE THREAD

**Issue by karolzlot** [theskumar/python-dotenv#443]
Are lists / arrays possible with python-dotenv?

Example:
```
TOKENS=["token1", "token2"]
ANOTHER_VARIABLE=5
```

**Comment by asowder3943** [comment]
It looks like this has been addressed in the past: #229 
.env file values are passed as strings; in your case, it is best to pass your configuration as a string and parse the list from that.

For example:
```
TOKENS="token1,token2"
```
Then in your python code:
```python
from dotenv import load_dotenv
import os
load_dotenv()
tokens_list = os.getenv('TOKENS').split(',')
```

**Comment by karolzlot** [comment]
ok, thanks
